src/components/subscriptions/data/service.js

```diff
@@ -46,6 +47,25 @@ function updateUserLicenseStatus({ userId, status }) {
   return null;
 }
 
+function updateUserRemindTimeStamp({ userId, bulkRemind, pendingSince }) {
```

Comment:
this may need to change slightly if we no longer pass in the `userId`, but rather the user's `emailAddress`, like what the remind/revoke endpoints will be expecting.

Reply:
Yes, this can be changed once we have a real endpoint.